### PR TITLE
Allow Raptor to compress data with ZSTD

### PR DESCRIPTION
**Pull request by highker**
On top of #12594

### CODE REVIEW

**Comment by highker** [comment]
Simple benchmark:
```
sf10 lineitem benchmark: `create table T1 from (select * from T2)`

Current ORC Writer + SNAPPY:	CPU Time: 384.7s, Wall Time 4:00 min, Compression Ratio: 28%
Optimized ORC Writer + SNAPPY:	CPU Time: 261.7s, Wall Time 2:49 min, Compression Ratio: 28%
Optimized ORC Writer + ZSTD:	CPU Time: 231.1s, Wall Time 2:09 min, Compression Ratio: 21%

**Comment by highker** [comment]
rebased

**Comment by highker** [comment]
Rebased

**Comment by highker** [comment]
I have turned it on for `TestOrcXXXwriter`. Wanna turn it on for query runner as well? @jessesleeping

File: presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcRecordWriter.java [hunk 1]
```diff
@@ -217,6 +221,20 @@ private static RecordWriter createRecordWriter(Path target, List<Long> columnIds
         }
     }
 
+    private static org.apache.hadoop.hive.ql.io.orc.CompressionKind toCompressionKind(CompressionKind compression)
+    {
+        switch (compression) {
+            case NONE:
```
Comment (jessesleeping) [inline presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcRecordWriter.java]: Do we want to ban using `NONE` as compression type?

**Pull request merged**